Add a new description for the DataGrid - Filter Panel demo (#78)
DevExpress/devextreme-demos: JSDemos/Demos/DataGrid/FilterPanel/description.md

> [Filter panel](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterPanel/) is a UI element that displays a combined filter. This filter is stored in the [filterValue](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterValue) option and consists of filters that users apply in other UI elemens ([filter row](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterRow/), [header filter](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/headerFilter/), [filterBuilder](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterBuilder)).

Reviewer: I"d say "the combined filter", because it the only one - the one that combines all others.

> [Filter panel](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterPanel/) is a UI element that displays the combined filter. This filter is stored in the [filterValue](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterValue) option and consists of filters that users apply in other UI elemens ([filter row](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterRow/), [header filter](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/headerFilter/), [filterBuilder](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterBuilder)).

Reviewer 2: ```suggestion
The [Filter panel](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterPanel/) is a UI element that displays the combined filter. This filter is stored in the [filterValue](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterValue) option and consists of filters that users apply in other UI elemens ([filter row](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterRow/), [header filter](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/headerFilter/), [filterBuilder](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterBuilder)).
```

To display the filter panel, set the **filterPanel**.[visible](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterPanel/#visible) option to **true**. 

> A click on the combined filter calls the integrated filter builder. You can configure it in the [filterBuilder](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterBuilder) object. In this demo, this object is used to add a [custom filter operation](/Documentation/ApiReference/UI_Widgets/dxFilterBuilder/Configuration/customOperations).

Reviewer: Why don't you mention what the custom operation is? You can be specific, so that it's easier for developers to find the code and try the functionality.

Author: @Abadzhev 
Sorry, I don't understand what you mean here. Should I add the value of the custom operation's `name` field? The fact is we have only one item in the `customOperations` array.
Or did I get you wrong, and I should describe the basics of custom filter operations? Users can do almost anything when they implement custom operations.

Reviewer: I meant "for developers to find the code and *for users* to try the functionality". Sorry it came out confusing. All I mean is you can mention what the operation is - what's it called, or what it does. Right now it sounds like "we added a new operation, look for it yourselves" :-)

> [Filter panel](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterPanel/) is a UI element that displays a filter expression. This expression consists of all filters that users apply in the [filter row](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterRow/) and [header filter](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/headerFilter/). A click on the filter expression calls the integrated filter builder. You can configure it in the [filterBuilder](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterBuilder) object. 
> 
> To display the filter panel, set the **filterPanel**.[visible](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterPanel/#visible) option to **true**. 
> 
> In this demo, we configure the `Weekends` [custom operation](/Documentation/ApiReference/UI_Widgets/dxFilterBuilder/Configuration/customOperations) that applies to data of the *"date"* [type](/Documentation/ApiReference/UI_Widgets/dxFilterBuilder/Configuration/customOperations/#dataTypes). The [calculateFilterExpression](/Documentation/ApiReference/UI_Widgets/dxFilterBuilder/Configuration/customOperations/#calculateFilterExpression) function returns the filter expression to be displayed in the filter panel. Users should not be able to edit this operation in UI, so the [hasValue](/Documentation/ApiReference/UI_Widgets/dxFilterBuilder/Configuration/customOperations/#hasValue) option is disabled.

Reviewer 3: ```suggestion
[Filter panel](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterPanel/) is a UI element that displays a combined filter. This filter is stored in the [filterValue](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterValue) option and consists of filters that users apply in other UI elemens ([filter row](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterRow/), [header filter](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/headerFilter/), [filterBuilder](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterBuilder)).

To display the filter panel, set the **filterPanel**.[visible](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterPanel/#visible) option to **true**. 

A click on the combined filter calls the integrated filter builder. You can configure it in the [filterBuilder](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterBuilder) object. In this demo, this object is used to add a [custom filter operation](/Documentation/ApiReference/UI_Widgets/dxFilterBuilder/Configuration/customOperations).
```

> The intergrated filter builder is displayed in a popup window. This is the DevExtreme [Popup](/Documentation/ApiReference/UI_Widgets/dxPopup/) component. Its default configuration is defined automatically, but you can redefine it in the [filterBuilderPopup](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterBuilderPopup) object. In this example, this object is used to specify the popup's [position](/Documentation/ApiReference/UI_Widgets/dxPopup/Configuration/#position).

Reviewer 2: ```suggestion
The intergrated filter builder is displayed in a pop-up window (the DevExtreme [Popup](/Documentation/ApiReference/UI_Widgets/dxPopup/) component). Its default configuration is defined automatically, but you can change it in the [filterBuilderPopup](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterBuilderPopup) object. In this example, this object is used to specify the [position](/Documentation/ApiReference/UI_Widgets/dxPopup/Configuration/#position) of the pop-up window.
```

> **DataGrid** uses the DevExtreme [Popup](/Documentation/ApiReference/UI_Widgets/dxPopup/) component to display a filter builder. The popup's default configuration is defined automatically. If this configuration does not meet your requirements, you can redefine it in the [filterBuilderPopup](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterBuilderPopup) option. In this example, this option is used to specify the popup's [position](/Documentation/ApiReference/UI_Widgets/dxPopup/Configuration/#position).

Reviewer 3: ```suggestion
The intergrated filter builder is displayed in a popup window. This is the DevExtreme [Popup](/Documentation/ApiReference/UI_Widgets/dxPopup/) component. Its default configuration is defined automatically, but you can redefine it in the [filterBuilderPopup](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterBuilderPopup) object. In this example, this object is used to specify the popup's [position](/Documentation/ApiReference/UI_Widgets/dxPopup/Configuration/#position).
```

> [Filter panel](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterPanel/) is a UI element that displays all filters that users apply in the [filter row](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterRow/) and [header filter](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/headerFilter/).

Reviewer 3: ```suggestion
The filter panel displays a filter combined from filters that users apply with UI elements, such as the [filter row](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterRow/) and [header filter](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/headerFilter/). A click on the combined filter invokes the integrated filter builder. This element allows users to change this filter.
```
